### PR TITLE
add file

### LINKED ISSUE #1: needs a first file

### PR DESCRIPTION
**Pull request by zramsay**
closes #1

### CODE REVIEW

File: moving-the-chains.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+Title: Moving The Chains to the Blockchain
+
+# Section 1
+
+## Section 1.1
```
Comment (samirpatel12) [inline moving-the-chains.md]: Lets name the section.

Comment (samirpatel12) [inline moving-the-chains.md]: Cool

Comment (samirpatel12) [inline moving-the-chains.md:5]: Cool

Comment (zramsay) [inline moving-the-chains.md:5]: not really...

**Pull request merged**